libs/blocks/tabs/tabs.css

```diff
@@ -264,6 +263,61 @@
   height: 24px;
 }
 
+.tabs .tab-paddles {
+  position: relative;
+  z-index: 3;
+}
+
+.tabs .tab-paddles .paddle {
+  position: absolute;
+  width: 32px;
+  height: 48px;
+  bottom: 0;
+  background: #fff;
+  border: 0;
+}
+
+.tabs .tab-paddles .paddle:disabled {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color);
+  background: var(--tabs-list-bg-gradient);
+}
+
+.tabs .tab-paddles .paddle-left {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color), 4px 0 4px -2px #0000001a;
+  left: 0;
+}
+
+.tabs .tab-paddles .paddle-right {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color), -4px 0 4px -2px #0000001a;
+  right: 0;
+}
+
+.tabs .tab-paddles .paddle::after {
+  content: "";
+  position: absolute;
+  top: 15px;
+  left: 9px;
+  right: 17px;
+  opacity: 100;
+  transition: opacity 150ms ease;
+}
+
+.tabs .tab-paddles .paddle:disabled::after {
+  opacity: 0;
+}
+
+.tabs .tab-paddles .paddle-left::after {
+  background: url("data:image/svg+xml,%3Csvg width='8' height='14' viewBox='0 0 8 14' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M6.5 0.750003C6.77979 0.750003 7.06006 0.856933 7.27344 1.0708C7.70069 1.49805 7.70069 2.19043 7.27344 2.61768L2.89063 7L7.27344 11.3823C7.70069 11.8096 7.70069 12.502 7.27344 12.9292C6.84668 13.3569 6.15332 13.3569 5.72656 12.9292L0.570308 7.77344C0.365228 7.56836 0.249998 7.29004 0.249998 7C0.249998 6.70996 0.365228 6.43164 0.570308 6.22656L5.72656 1.0708C5.93994 0.856933 6.22021 0.750493 6.5 0.750003Z' fill='%236D6D6D'/%3E%3C/svg%3E%0A") no-repeat center center;
```

Comment:
If your just using a `<img src'">` asset. Can we just reference the [UI icons from here](https://adobe.sharepoint.com/sites/adobecom/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2Fadobecom%2FShared%20Documents%2Fmilo%2Flibs%2Fimg%2Fui&viewid=d776cf70%2D9b7e%2D4ab7%2Db9da%2D9e0f8e03a7d2)? 
`url('./libs/img/ui/...)` - That way all block can reference the same icon and updated or changes can be made in authoring. Otherwise we may want to include an inline SVG so they can be styled: hover etc.

Reply:
Updated this to an inline SVG.